src/components/TopMenu.js

```diff
@@ -44,20 +71,27 @@ const TopMenu = () => {
         <div className="flex items-center">
           <ul className="hidden md:flex uppercase font-semibold ">
             {navItems.map(({ href, label }) => (
-              
-                <NavItem
-                  key={href}
-                  href={href}
-                  onClick={handleSmallerScreensNavigation}
-                >
-                  {label}
-                </NavItem>
-              
+
+              <NavItem
+                key={href}
+                href={href}
+                onClick={handleSmallerScreensNavigation}
+              >
+                {label}
+              </NavItem>
+
             ))}
           </ul>
           {isLoggedIn ? (
```

Comment:
is really needed the `isLoggedIn` flag maybe just we can put `tokenDataEmail ?`. what do you think?

Reply:
I'm gonna check it